scripts/post-remove.sh

```diff
@@ -15,28 +15,40 @@ function disable_chkconfig {
     rm -f /etc/init.d/telegraf
 }
 
-if [[ "$1" == "0" ]]; then
-    # RHEL and any distribution that follow RHEL, Amazon Linux covered
-    # telegraf is no longer installed, remove from init system
-    rm -f /etc/default/telegraf
+if [[ -f /etc/redhat-release ]]; then
+    # RHEL-variant logic
+    if [[ "$1" = "0" ]]; then
+        # InfluxDB is no longer installed, remove from init system
+        rm -f /etc/default/telegraf
 
-    which systemctl &>/dev/null
-    if [[ $? -eq 0 ]]; then
-        disable_systemd
-    else
-        # Assuming sysv
-        disable_chkconfig
+        if pidof systemd &>/dev/null; then
+            disable_systemd
+        else
+            # Assuming sysv
+            disable_chkconfig
+        fi
     fi
-elif [ "$1" == "remove" -o "$1" == "purge" ]; then
+elif [[ -f /etc/lsb-release ]]; then
     # Debian/Ubuntu logic
```

Comment:
How come in `post-install.sh`, the debian/ubuntu logic is controlled by a test for `/etc/debian_version`, but here it's a test for `/etc/lsb-release`?